src/test/java/AccountTest.java

```diff
@@ -103,6 +104,27 @@ public void testConstructorFromJson() {
     assertEquals("Failed to construct Account object from JSON", expectedResponse, actualResponse);
   }
 
+  /** Test that getAccountSheetHeader returns the correct Account headings. */
+  @Test
+  public void testGetAccountSheetHeader() {
```

Comment:
Not really sure that this test adds much value, since it's effectively a change detector.
But it's fine with me to include it